C13.csv

```diff
@@ -44,6 +44,12 @@ CodeFigure_DataCategories,Name_DataCategories_en,CodeFigure_InternationalDataSub
 2,Vertical soundings (other than satellite),25,Climatological observations from fixed-land stations (CLIMAT TEMP),Operational
 2,Vertical soundings (other than satellite),26,Climatological observations from ships (CLIMAT TEMP SHIP),Operational
 2,Vertical soundings (other than satellite),40,MW temperature/humidity sounding,Operational
+2,Vertical soundings (other than satellite),104,Upper-level temperature/humidity/wind reports from fixed-land stations (TEMP) with re-used radiosondes,Operational
+2,Vertical soundings (other than satellite),105,Upper-level temperature/humidity/wind reports from ships (TEMP SHIP) with re-used radiosondes,Operational
+2,Vertical soundings (other than satellite),106,Upper-level temperature/humidity/wind report from mobile land stations (TEMP MOBIL) with re-used radiosondes,Operational
+2,Vertical soundings (other than satellite),114,Upper-level temperature/humidity/wind reports from descent radiosondes originally launched from fixed-land stations with re-used radiosondes,Operational
```

Comment:
@SibylleK Did you intentionally skip 109-113?

Reply:
@amilan17 Yes, as 100 is added to the existing sub-categories 4,5,6 and 14 if a radiosonde is re-used.